converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/RetryableFhirTransactionPipeline.cs

```diff
@@ -21,24 +20,11 @@ namespace Microsoft.Health.DicomCast.Core.Features.Worker.FhirTransaction
     /// </summary>
     public class RetryableFhirTransactionPipeline : IFhirTransactionPipeline
     {
-        private const int MaxRetryCount = 3;
-
         private readonly IFhirTransactionPipeline _fhirTransactionPipeline;
         private readonly IExceptionStore _exceptionStore;
         private readonly IAsyncPolicy _retryPolicy;
 
-        public RetryableFhirTransactionPipeline(IFhirTransactionPipeline fhirTransactionPipeline, IExceptionStore exceptionStore)
-            : this(
-                  fhirTransactionPipeline,
-                  exceptionStore,
-                  MaxRetryCount)
-        {
-        }
-
-        internal RetryableFhirTransactionPipeline(
-            IFhirTransactionPipeline fhirTransactionPipeline,
-            IExceptionStore exceptionStore,
-            int maxRetryCount)
+        public RetryableFhirTransactionPipeline(IFhirTransactionPipeline fhirTransactionPipeline, IExceptionStore exceptionStore, int maxRetryCount = 15)
```

Comment:
instead of max retry county, we pass in a timespan?

Reply:
Changed it to max of 10 mins by default